File: tests/python/dict_hash/key_exists.py
from os import environ

dic = {}
dic[environ['arr1']] = ""
dic[environ['arr2']] = ""
dic[environ['arr3']] = ""
test1 = environ['test1']
test2 = environ['test2']

print(str(test1 in dic).lower())
print(str(test2 in dic).lower())

File: tests/nodejs/dict_hash/key_exists.js
const obj = {}
obj[process.env.arr1] = ""
obj[process.env.arr2] = ""
obj[process.env.arr3] = ""

const test1 = process.env.test1
const test2 = process.env.test2
console.log(test1 in obj)
console.log(test2 in obj)
